index.html

```diff
@@ -184,7 +184,13 @@ <h3 id="the-difficulties-of-ruby-processing">The difficulties of ruby processing
 case by case as was done in movable type typography.
 It is therefore necessary to decide upon comprehensive rules
 that provide solutions to all the problems listed above,
-so that placement may be determined fully automatically.</p>
+so that placement may be determined fully automatically. 
+The system to be designed needs to be very complex, 
+like ones described in <a href="https://w3c.github.io/jlreq/#positioning_of_jukugoruby">Positioning of Jukugo-ruby</a> in [[?JLREQ]], 
```

Comment:
```suggestion
like ones described in <a href="https://www.w3.org/TR/jlreq/#positioning_of_jukugoruby">Positioning of Jukugo-ruby</a> in [[?JLREQ]], 
```

Might be better to link to the /TR/ URL.

Reply:
Surely, really thank you!!!